src/components/DistBuilder.re

```diff
@@ -142,12 +142,15 @@ module DemoDist = {
                },
                ~distPlusIngredients,
                ~shouldDownsample=options.downsampleTo |> E.O.isSome,
-               ~recommendedLength=options.downsampleTo |> E.O.default(10000),
+               ~recommendedLength=options.downsampleTo |> E.O.default(100),
```

Comment:
Was this changed intentionally? Downsampling is a very specific part of the codebase, so at this point i'm much more worried that the rest of it works well.

Reply:
No, sorry – I had messed with that to find out which one of the defaults kicks in when the downsampling text field is left blank. Nothing to see here.